config/configauth/README.md

```diff
@@ -17,6 +25,20 @@ extensions:
     issuer_url: http://localhost:8080/auth/realms/opentelemetry
     audience: account
 
+  oauth2client:
```

Comment:
Did you try those examples?

Reply:
yes @jpkrohling  this is from the E2E test we carried out  https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/3848#issue-926422174 
Works fine.